app/views/asset/show_hwdetails.scala.html

```diff
@@ -73,6 +73,23 @@
     </tbody>
   </table>
 
+  <h4>GPU <small>Collected GPU Information</small></h4>
+  <table class="table table-bordered table-hover table-condensed">
+    <thead>
+      <tr>
+        <th>Id</th><th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+    @aa.lshw.gpus.zipWithIndex.map { case(gpu,id) =>
+      <tr>
+        <th>@id</th>
+        <td>@gpu.description</td>
```

Comment:
add vendor column here, and maybe sort by id, so the GPUs show up in enumeration order?